### PR TITLE
Refactor clusters deployment

### PR DESCRIPTION
**Pull request by eth2devops**
- Update the deployment scripts to get a list of VCs and charon versions from the config backend of the cluster and deploy the k8s templates.
- Merge the canary and regular deployments in one script
- Remove the relay and bootnodes configuration, and let it use the charon defaults
- Update the CD workflows with the new scripts names

Ticket #73

### CODE REVIEW

File: voluntary-exit.sh [hunk 1]
```diff
@@ -1,72 +0,0 @@
-#!/bin/bash
```
Comment (LukeHackett12) [inline voluntary-exit.sh:1]: Does this no longer apply?

Comment (eth2devops) [reply voluntary-exit.sh:1]: This script turned obsolete as it only exits the teku vc. I will create a new story to add exit templates for the other VCs (LH, lodester, and vouch).

Comment (LukeHackett12) [reply voluntary-exit.sh:1]: Ah ok cool 👌

**Review by LukeHackett12 — APPROVED** [review]
LGTM

**Pull request merged**